WordPress/src/main/java/org/wordpress/android/ui/plans/PlansActivity.java

```diff
@@ -98,6 +102,18 @@ public void onClick(View v) {
         } else {
             setupPlansUI();
         }
+
+        // navigate to the "manage plans" page for this blog when the user clicks the manage bar
+        mManageBar.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                Blog blog = WordPress.getBlog(mLocalBlogID);
+                if (blog == null) return;
```

Comment:
It's funny you mention that because I originally did that when the blog was null, but it seemed like such an edge case (since we check for a null blog in `onCreate()`) that I deleted it. I think we're fine without it.